compiler/x/i386/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1704,6 +1704,12 @@ OMR::X86::I386::TreeEvaluator::vdcmpleEvaluator(TR::Node *node, TR::CodeGenerato
    }
 
 
+TR::Register*
+OMR::X86::I386::TreeEvaluator::vfmaEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
```

Comment:
I think I just follow an example of vdsqrt. In which class should I define vfma so that it works for both? @BradleyWood

Reply:
I was just referring to commoning it in compiler/x/codegen/OMRTreeEvaluator

Reply:
I think the evaluator needs to be defined in both inOMR::X86::AMD64::TreeEvaluator and in OMR::X86::I386::TreeEvaluator classes to avoid linkage errors. Currently they are both  unimplemented. I think it's better to do the commoning when they are actually implemented. This PR is just an  example how to add a new vector opcode.